### PR TITLE
Create category page

### PR DESCRIPTION
**Pull request by QianyingLuo**

### CODE REVIEW

File: index commented.txt
Comment (devs-toni) [inline index commented.txt:1]: please remove this file

Comment (QianyingLuo) [reply index commented.txt:1]: Done

File: src/controller/category.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+@router.get("/laptops", response_class=HTMLResponse)
+def render_laptops(request: Request): 
+    laptops = product_api.get_products_by_type("portatil")
+    product_type = "Portátiles"
+    return templates.TemplateResponse("pages/category.html", {"request": request, "products": laptops, "type": product_type})
+
+@router.get("/smartphones", response_class=HTMLResponse)
+def render_laptops(request: Request): 
+    smartphones = product_api.get_products_by_type("smartphone")
```
Comment (devs-toni) [inline src/controller/category.py]: Change the functions naming

**Review by devs-toni — CHANGES_REQUESTED** [review]

File: src/controller/category.py (hunk 1 above)
Comment (QianyingLuo) [reply src/controller/category.py]: Done

**Review by devs-toni — APPROVED** [review]

**Pull request merged**